images/linux/toolsets/toolset-1604.json

```diff
@@ -120,19 +120,13 @@
     "azureModules": [
         {
             "name": "az",
+            "url" : "https://raw.githubusercontent.com/Azure/az-ps-module-versions/main/versions-manifest.json",
             "versions": [
-                "1.0.0",
-                "1.6.0",
-                "2.3.2",
-                "2.6.0",
-                "2.8.0",
+                "5.5.0"
+            ],
+            "zipversions": [
                 "3.1.0",
```

Comment:
Why don't we zip the same versions for all images?

U16 - 3.8.0, 4.8.0
U18 - 3.8.0, 4.8.0
U20 - 3.8.0, 4.8.0

Reply:
We checked usage for each module on each image. since 3.1.0 and 4.4.0 were never been made available on ubuntu20 its usage is zero. so we don't want to add those versions on U20. 

for U16 and U18 3.1.0 and 4.4.0 are mostly used versions. for other versions which are rarely used we are modifying task which use these modules to download it on the run.

This is to reduce disk space

I have updated description to get more context.